addon/components/bs-alert.js

```diff
@@ -92,14 +85,15 @@ export default class Alert extends Component {
    * @default true
    * @public
    */
-  @arg
-  visible = true;
+  get visible() {
+    return this._visible ?? this.args.visible ?? true;
+  }
 
   /**
    * @property _visible
    * @private
    */
-  @localCopy('visible')
+  @tracked
   _visible;
```

Comment:
I wonder if this still works as expected. `localCopy` observes changes to the source and resets the local changes whenever source value changes:

> @localCopy
>
> Creates a local copy of a remote value. The local copy can be updated locally, but will also update if the remote value ever changes
>
> https://github.com/tracked-tools/tracked-toolbox#localcopy

Let's double check that we have sufficient test coverage for the scenario that `@visible` argument changes _after_ the internal visibility changed.